share/instruments/instruments.xml

```diff
@@ -1001,6 +1001,7 @@
                   <Channel>
                         <program value="68"/>
                   </Channel>
+				  <genre>orchestra</genre>
```

Comment:
Something is wrong with mixing whitespaces/tabs. We prefer whitespaces everywhere.

Reply:
Tabs are whitespace ;-). We prefer spaces (which are also whitespace)